### ISSUE TITLE
Error in function call for NegativeBinomial in documentation for crafting vanilla pytorch model

### ISSUE THREAD

**Issue by rangan2510** [YosefLab/scvi-tools#1297]
I am trying to create my own PyTorch module by looking at the documentation as in [Crafting the Module in vanilla PyTorch](https://docs.scvi-tools.org/en/stable/tutorials/notebooks/module_user_guide.html#Crafting-the-Module-in-vanilla-PyTorch). In the next step, I want to implement the model as in [Constructing a high-level model](https://docs.scvi-tools.org/en/stable/tutorials/notebooks/model_user_guide.html). 

In the first linked page, the `loss` function has a line 
`log_lik = NegativeBinomial(total_counts=theta, total=nb_logits).log_prob(x).sum(dim=-1)`

whereas the [PyTorch documentation](https://pytorch.org/docs/1.2.0/distributions.html#torch.distributions.negative_binomial.NegativeBinomial) for the function is given as
`torch.distributions.negative_binomial.NegativeBinomial(total_count, probs=None, logits=None, validate_args=None)` 

So, in the next stage, when I am replacing the built-in `VAE` model with `MyModule`, I am getting the `unexpected keyword argument 'total_counts'` error.

I am attaching the file for reproducing the issue. 

[scvi_tut.py.txt](https://github.com/YosefLab/scvi-tools/files/7789848/scvi_tut.py.txt)

I am using PyTorch 1.10.0+cpu.

**Comment by PierreBoyeau** [comment]
We use a custom `NegativeBinomial` distribution object, inheriting from torch's `Distribution`. 
The reason is because most of our models use the mean/overdispersion parameterization, that is currently not handled by Pytorch.

If you want to use our NB implementaition in your own model, use `from scvi.distributions import NegativeBinomial`.

**Comment by adamgayoso** [comment]
@PierreBoyeau I think it's because Romain used the default pytorch one here:

https://docs.scvi-tools.org/en/stable/tutorials/notebooks/module_user_guide.html#Crafting-the-Module-in-vanilla-PyTorch

Can you check that it's correct?

**Comment by adamgayoso** [comment]
Closing now due to tutorials fix
